content/docs/reference/types/object.mdx

```diff
@@ -45,15 +45,15 @@ Ultimately there are 3 ways to configure this type...
 
 * With `list` set to `false`, the `fields` array can be used to wrap some data – this appears in the editor as a nested page. This can be used to organise the CMS, such as grouping options together, as in the gif below.
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331124/tina-io/docs/Sep-26-2024_16-05-37_d2ol8w.gif)
+<WebmEmbed embedSrc="/img/docs/reference/rodmap-grid.webm"/>
```

Comment:
**issue**: The quality of this image and those below it is very poor - they either need to be re-encoded or shrunk.

![image](https://github.com/user-attachments/assets/8babaae5-af0a-4277-b1e5-9e0aa2582801)
**Figure: The image is pixelated so badly that the point of the animation is lost**

Reply:
We have added the option for users to specify the width. for these webm's we use 50% width so we shrink them into a better resolution